codegens/objective-c/lib/index.js

```diff
@@ -0,0 +1,285 @@
+var _ = require('./lodash'),
+  sanitizeOptions = require('./util').sanitizeOptions,
+  sanitize = require('./util').sanitize,
+  self;
+
+/**
+ * Parses Raw data to fetch syntax
+ *
+ * @param {Object} body Raw body data
+ * @param {String} indent Indent
+ * @param {Boolean} trim Trim request body if this is true.
+ */
+function parseRawBody (body, indent, trim) {
+  var bodySnippet = '';
+  bodySnippet += 'NSData *postData = [[NSData alloc] initWithData:[@"' + sanitize(body.toString(), trim) + '" ' +
+  'dataUsingEncoding:NSUTF8StringEncoding]];\n';
+  bodySnippet += '[request setHTTPBody:postData];\n';
+  return bodySnippet;
+}
+
+/**
+ * Parses GraphQL body
+ *
+ * @param {Object} body GraphQL body
+ * @param {String} indent Indent
+ * @param {Boolean} trim Trim request body if this is true.
+ */
+function parseGraphqlBody (body, indent, trim) {
```

Comment:
can we change this to parseGraphQLBody? or just parseGraphQL? We have used parseGraphQL everywhere else.

Reply:
Done